labelbox/pagination.py

```diff
@@ -131,10 +131,10 @@ def get_next_page(self) -> Tuple[Dict[str, Any], bool]:
 
 class _CursorPagination(_Pagination):
 
-    def __init__(self, cursor_path: List[str], *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(self, cursor_path: List[str], params={}, *args, **kwargs):
+        super().__init__(*args, **dict(kwargs, params=params))
```

Comment:
might be shorter to just use ```{**kwargs, **params}```
but I think this is getting weirder, it's probably better to leave it as it was before.